cli/cmd/network.go

```diff
@@ -0,0 +1,171 @@
+package cmd
+
+import (
+	"fmt"
+	"sort"
+
+	"github.com/AlecAivazis/survey/v2"
+	"github.com/spf13/cobra"
+
+	"github.com/oasisprotocol/oasis-sdk/cli/cmd/common"
+	"github.com/oasisprotocol/oasis-sdk/cli/config"
+	"github.com/oasisprotocol/oasis-sdk/cli/table"
+)
+
+var (
+	networkCmd = &cobra.Command{
+		Use:   "network",
+		Short: "Manage network endpoints",
+	}
+
+	networkListCmd = &cobra.Command{
+		Use:     "list",
+		Aliases: []string{"ls"},
+		Short:   "List configured networks",
+		Run: func(cmd *cobra.Command, args []string) {
+			cfg := config.Global()
+			table := table.New()
+			table.SetHeader([]string{"Name", "Chain Context", "RPC"})
+
+			var output [][]string
+			for name, net := range cfg.Networks.All {
+				output = append(output, []string{
+					name,
+					net.ChainContext,
+					net.RPC,
+				})
+			}
+
+			// Sort output by name.
+			sort.Slice(output, func(i, j int) bool {
+				return output[i][0] < output[j][0]
+			})
+
+			table.AppendBulk(output)
+			table.Render()
+		},
+	}
+
+	networkAddCmd = &cobra.Command{
+		Use:   "add <name> <chain-context> <rpc-endpoint>",
+		Short: "Add a new network",
+		Args:  cobra.ExactArgs(3),
+		Run: func(cmd *cobra.Command, args []string) {
+			cfg := config.Global()
+			name, chainContext, rpc := args[0], args[1], args[2]
+
+			// Ask user for some additional parameters.
+			questions := []*survey.Question{
+				{
+					Name:   "description",
+					Prompt: &survey.Input{Message: "Description:"},
```

Comment:
There is a bit of an inconsistency. In the `oasis network list` we use `NAME` and here we ask for `Description`.
I'm slightly more in favor of using `Name` here and perhaps `network_name` for the variables to avoid confusion with all other `Name` things in this part of the code :slightly_smiling_face:

Reply:
So there are two things, "name" and "description". The name is an identifier used in various places to reference networks, paratimes etc. while "description" is something human readable that is also shown when signing a transaction.